setup/build_dagmc.sh

```diff
@@ -0,0 +1,298 @@
+#!/bin/bash
+
+# Get compilers and set up paths
+function get_compile() {
+  cd $base_dir
+  wget --spider http://proxy.chtc.wisc.edu/SQUID/$username/$compile_tar
+  if [ $? == 0 ]; then
+    wget http://proxy.chtc.wisc.edu/SQUID/$username/$compile_tar
+  else
+    echo $compile_tar not found
+    exit
+  fi
+  tar -xzvf $compile_tar
+  export PATH=$compile_dir/gcc/bin:$PATH
+  export PATH=$compile_dir/openmpi/bin:$PATH
+  export LD_LIBRARY_PATH=$compile_dir/gmp/lib:$LD_LIBRARY_PATH
+  export LD_LIBRARY_PATH=$compile_dir/mpfr/lib:$LD_LIBRARY_PATH
+  export LD_LIBRARY_PATH=$compile_dir/mpc/lib:$LD_LIBRARY_PATH
+  export LD_LIBRARY_PATH=$compile_dir/gcc/lib:$LD_LIBRARY_PATH
+  export LD_LIBRARY_PATH=$compile_dir/gcc/lib64:$LD_LIBRARY_PATH
+  export LD_LIBRARY_PATH=$compile_dir/openmpi/lib:$LD_LIBRARY_PATH
+}
+
+# Build HDF5
+function build_hdf5() {
+  cd $build_dir
+  mkdir -p hdf5/bld
+  cd hdf5
+  hdf5_tar=hdf5-$hdf5_version.tar.gz
+  wget --spider http://proxy.chtc.wisc.edu/SQUID/$username/$hdf5_tar
+  if [ $? == 0 ]; then
+    wget http://proxy.chtc.wisc.edu/SQUID/$username/$hdf5_tar
+  else
+    wget https://www.hdfgroup.org/ftp/HDF5/releases/hdf5-$hdf5_version/src/$hdf5_tar
+  fi
+  tar -xzvf $hdf5_tar
+  ln -s hdf5-$hdf5_version src
+  cd bld
+  config_string=
+  config_string+=" "--enable-shared
+  config_string+=" "--disable-debug
+  config_string+=" "--prefix=$dagmc_dir/hdf5
+  ../src/configure $config_string
+  make -j $jobs  # j=12: 1:02.21 wall time, 307884 kB mem
+  make install
+  export PATH=$dagmc_dir/hdf5/bin:$PATH
+  export LD_LIBRARY_PATH=$dagmc_dir/hdf5/lib:$LD_LIBRARY_PATH
+  cd $base_dir
+}
+
+# Build CUBIT
+function build_cubit() {
```

Comment:
In @ljacobson64's case he does since part of the DAGMC testing requires that we may want to test against using sat files directly.